traefik/plan.sh

```diff
@@ -0,0 +1,78 @@
+pkg_name=traefik
+pkg_description="a modern reverse proxy"
+pkg_upstream_url="https://traefik.io"
+pkg_origin=core
+pkg_version="v1.3.6"
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("MIT")
+pkg_source="http://github.com/containous/traefik"
+pkg_build_deps=(
+  core/node
+  core/sed
+  core/yarn
+)
+pkg_deps=()
+pkg_bin_dirs=(bin)
+pkg_svc_user="root"
+pkg_svc_group="root"
+pkg_scaffolding=core/scaffolding-go
+scaffolding_go_base_path=github.com/containous
+scaffolding_go_build_deps=()
+
+pkg_exports=(
+  [web_port]=web.port
+  [web_host]=web.host
+  [web_enabled]=web.enable
+)
+
+do_prepare() {
+  build_line "adding \$GOPATH/bin to \$PATH"
+  export PATH=${scaffolding_go_gopath:?}/bin:$PATH
+
+  build_line "building go-bindata"
+  go get github.com/jteeuwen/go-bindata
+  go install github.com/jteeuwen/go-bindata/...
```

Comment:
I'm not sure I've understood the _static linked binary_ question. Is this something we want to do? Is it a nice to have? I feel like I've missed the boat there 😄